packages/react-server/src/logger/logger.ts

```diff
@@ -2,7 +2,7 @@ import {Context} from 'koa';
 import chalk from 'chalk';
 import {KoaNextFunction} from '../types';
 
-const LOGGER = Symbol('logger');
+export const LOGGER = Symbol('logger');
```

Comment:
Lets keep the `Symbol` private to this module and instead create a `setLogger()` and export that. It is more consistent with how we've implemented middleware in the past.

Have a look at the example here: https://github.com/Shopify/web/blob/master/server/middleware/handler/handler.ts#L18